docs/proposals/volumes.md

```diff
@@ -384,16 +401,15 @@ type Kubelet struct {
 }
 
 func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap, error) {
-    podFSGroup = pod.Spec.PodSecurityContext.FSGroup
-    podFSGroupSet := false
-    if podFSGroup != 0 {
-        podFSGroupSet = true
-    }
-
     podVolumes := make(kubecontainer.VolumeMap)
-
     for i := range pod.Spec.Volumes {
         volSpec := &pod.Spec.Volumes[i]
+        hasFSGroup := false
```

Comment:
this whole block looks const in the loop (e.g. I don't see `i` anywhere in the block) why is it moved into the loop?